internal/cmd/auth.go

```diff
@@ -2,44 +2,72 @@ package cmd
 
 import (
 	"context"
+	"fmt"
+	"os"
 
 	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 	"github.com/stateful/runme/internal/auth"
 	"github.com/stateful/runme/internal/tui"
 )
 
-func authCmd() *cobra.Command {
+func loginCmd() *cobra.Command {
 	cmd := &cobra.Command{
-		Hidden: true,
-		Use:    "auth",
-		Short:  "Log in and out of your Stateful",
+		Use:   "login",
+		Short: "Log in to Runme",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return newAuth().Login(cmd.Context())
+		},
 	}
 
-	cmd.AddCommand(loginCmd())
-	cmd.AddCommand(logoutCmd())
-
 	return cmd
 }
 
-func loginCmd() *cobra.Command {
+func logoutCmd() *cobra.Command {
 	cmd := &cobra.Command{
-		Use: "login",
+		Use:   "logout",
+		Short: "Log out from Runme",
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return newAuth().Login(cmd.Context())
+			return newAuth().Logout()
 		},
 	}
-
 	return cmd
 }
 
-func logoutCmd() *cobra.Command {
+func tokenCmd() *cobra.Command {
+	var insecureF bool
+
 	cmd := &cobra.Command{
-		Use: "logout",
+		Use:    "token",
+		Hidden: true,
+		Short:  "Print runme API token",
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return newAuth().Logout()
+			var token string
+			var err error
+			if insecureF {
+				auth := newAuth()
+				token, err = auth.GetToken(cmd.Context())
+				if err != nil {
+
+					if err := checkAuthenticated(cmd.Context(), cmd, auth, !recoverableWithLogin(err)); err != nil {
+						return err
+					}
+
+					token, err = auth.GetToken(cmd.Context())
+					if err != nil {
+						return err
+					}
+				}
+				_, err = fmt.Fprint(os.Stdout, token, "\n")
+				return err
+			}
+			_, err = fmt.Fprint(os.Stdout, "To use this command, please add the --insecure parameter")
```

Comment:
```suggestion
			return errors.New("To use this command, please add the --insecure parameter")
```

As mentioned in the other comment, you generally want to avoid using `os.Stdout` directly and you also generally want to discard errors from stdout.

In this case, we should definitely return an error with the message directly.